Tutorial 3 axel branch pull request

hauduc opened the pull request:
Hi Stephan,

Please find the updated Tutorial 3 content ready to go. Would you be able to try the PDF displaying strategy I sent you last week to see if it works on the website? It also failed on the RStudio html preview for me, but it worked when I viewed the bookdown html on my browser, so I figure than maybe once the bookdown deploys to the website, it will be visible too!

Best,
Axel

stephan-koenig commented:
Hi Axel,

Did you delete your local branch and start a new one as I described to you? As you can see from your commit history, it still contains all the commits you have done before, and that makes it very difficult to resolve your pull request. I will resolve this manually and then delete the old `axel` branch on GitHub. If you need help fixing your local repo, let me know, and I can walk you through it on Zoom (even today if you have time before 5 pm).

Best,

Stephan

hauduc commented:
Hi Stephan,

Sorry for the delay and thank you for setting up the new tutorial webpage and making the edits to make the links look nice!

Ok, I think I deleted my branch:

***@***.*** MINGW64 ~/Documents/github/MICB405 (main)
$ git branch -D axel
Deleted branch axel (was c611e47).

***@***.*** MINGW64 ~/Documents/github/MICB405 (main)
$ git branch -d axel
error: branch 'axel' not found.

>>> C:/Program Files/Git/bin/git.exe checkout axel
Branch 'axel' set up to track remote branch 'axel' from 'origin'.
Switched to a new branch 'axel'

Hopefully it won't push old commits now.

Best,
Axel
________________________________
From: Stephan Koenig ***@***.***>
Sent: Thursday, September 23, 2021 16:08
To: EDUCE-UBC/MICB405 ***@***.***>
Cc: Axel Hauduc ***@***.***>; Author ***@***.***>
Subject: Re: [EDUCE-UBC/MICB405] Tutorial 3 axel branch pull request (#17)


Closed #17<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FEDUCE-UBC%2FMICB405%2Fpull%2F17&data=04%7C01%7C%7C5bc644105b5c4115ac6d08d97ee70b62%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637680353059342347%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Qo8M%2B1afSs6%2BQemDBoulSX6NcAv8kLkUqgHEsqlcjEg%3D&reserved=0>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FEDUCE-UBC%2FMICB405%2Fpull%2F17%23event-5352874579&data=04%7C01%7C%7C5bc644105b5c4115ac6d08d97ee70b62%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637680353059352341%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=szsktLWbFgqmWy%2BI8QkyCRD9ihvFE2KYeaBPC0yBlko%3D&reserved=0>, or unsubscribe<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FANMIURWWNO3PE64W3UIYKATUDOXOPANCNFSM5EUWM55A&data=04%7C01%7C%7C5bc644105b5c4115ac6d08d97ee70b62%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637680353059352341%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=0mvhG0d%2BgHZFEsgSvqFJ2dSuZ%2B5TgZdTJsijaIjrP1o%3D&reserved=0>.

hauduc commented:
Ok, it looks like I can also delete my branch on GitHub - just did it here. Hopefully now we should be good.

Best,
Axel
